.travis.yml

```diff
@@ -26,8 +26,16 @@ env:
 before_install:
   # See https://github.com/travis-ci/travis-ci/issues/4629#issuecomment-239493916
   - rm ~/.m2/settings.xml
-  - curl -Ls https://git.io/jabba | bash && . ~/.jabba/jabba.sh
-install: jabba install $(jabba ls-remote "adopt@~1.$TRAVIS_JDK.0-0" --latest=patch) && jabba use "$_" && java -Xmx32m -version
+  - curl -s "https://get.sdkman.io" | bash
+  - source "$HOME/.sdkman/bin/sdkman-init.sh"
+  - echo sdkman_auto_answer=true > ~/.sdkman/etc/config
+  # Just to debug the installation
+  - sdk version
+install:
+  - sdk install java $(sdk list java | grep -o "$TRAVIS_JDK\.[0-9]*\.[0-9]*\.hs-adpt" | head -1) || true
```

Comment:
What do you think about using a build that passed the JCK instead of AdoptOpenJDK?  [Liberica](https://sdkman.io/jdks#Bellsoft) as example.

Reply:
ATM, Lightbend only certifies AdoptOpenJDK (and RH JDK) to run its stack (https://developer.lightbend.com/docs/introduction/getting-help/java-versions.html)